docs/user/advanced.rst

```diff
@@ -782,7 +782,7 @@ is to POST to the thread. Let's do it.
 ::
 
     >>> body = json.dumps({u"body": u"Sounds great! I'll get right on it!"})
-    >>> url = u"https://api.github.com/repos/psf/requests/issues/482/comments"
+    >>> url = u'https://api.github.com/repos/psf/requests/issues/482/comments'
```

Comment:
Good afternoon. Why change the type of quotation marks?

Reply:
> Good afternoon. Why change the type of quotation marks?

Since the contribution guide (https://github.com/psf/requests/blob/master/docs/dev/contributing.rst) states to use single-quoted strings where possible. Also I thought it would be a simple first PR.. :)